python/lsst/ap/association/diaPipe.py

```diff
@@ -122,6 +122,11 @@ class DiaPipelineConfig(pipeBase.PipelineTaskConfig,
         target=PackageAlertsTask,
         doc="Subtask for packaging Ap data into alerts.",
     )
+    doSerializeAlerts = pexConfig.Field(
```

Comment:
Is `doSerializeAlerts` the write term here?

I mean, it's correct, but it seems to add an unnecessary third term beyond “package” (on the basis that it's `PackageAlertsTask`, rather than `SerializeAlertsTask` that we're referring to) and “write” (on the basis that it's really writing out the alerts that the user cares about)

It's not a big deal, it just feels like we're using three terms which all kind-of-but-not-quite mean the same thing, where one (or _maybe_ two) would do fine.

Reply:
Changed everything to PackageAlerts.